test/PositionContext.test.ts

```diff
@@ -451,14 +451,37 @@ suite("PositionContext", () => {
         });
     });
 
-    suite("completionItems", () => {
+    suite("completionItems", async () => {
+        function addCursor(documentText: string, markerIndex: number): string {
+            return `${documentText.slice(0, markerIndex)}<CURSOR>${documentText.slice(markerIndex)}`;
+        }
+
         function completionItemsTest(documentText: string, index: number, expectedCompletionItems: Completion.Item[]): void {
-            test(`with ${Utilities.escapeAndQuote(documentText)} at index ${index}`, () => {
+            const testName = `with ${Utilities.escapeAndQuote(addCursor(documentText, index))} at index ${index}`;
+            test(testName, async () => {
+                let keepInClosureForEasierDebugging = testName;
+                keepInClosureForEasierDebugging = keepInClosureForEasierDebugging;
```

Comment:
Intentional to check in this and below commented-out code?

Reply:
1 yes, 2 no